app/models/manageiq/providers/redhat/infra_manager/ovirt_services/strategies/v4.rb

```diff
@@ -458,6 +456,19 @@ def build_template_from_hash(args)
         }.compact
         OvirtSDK4::Template.new(options)
       end
+
+      def platform
+        vm = get
+        vm.os.type
+      end
+
+      def update_sysprep!(content)
```

Comment:
Will this `raise` if it fails?

Reply:
It could raise sdk communication error. Will fix.

Reply:
The error will be propagated to the user so it is correct way of "not" handling it here.